gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Perform a case-insensitive comparison when filtering sensitive parameters from logs
```

Comment:
Could you mark this as an `* Issue` instead of a `* Feature` ? I'd classify this as a bug.

Reply:
For sure!